frontend/src/employees/DetailEmployee.tsx

```diff
@@ -0,0 +1,40 @@
+import {Employee} from "./Employee";
+import {useParams} from "react-router-dom";
+import {useState} from "react";
+
+type DetailEmployeesProps = {
+    employees: Employee[],
+    employeeUpdate: (employee: Employee) => void,
+}
+export default function DetailEmployee(props: DetailEmployeesProps) {
```

Comment:
Einige Bemerkungen zur aktuellen Verhalten von Details Komponente:
1) beim ersten speichern wird die ganze Seite neu geladen obwohl man nur emloyees laden sollte, dazu dementsprechend sind die input Felder geleert (beim zweiten speichern ist alles ok)
2) nach dem ersten speichern wird "?" zur url zugefügt (url: http://localhost:3000/?#/employees/9fa85530-3c48-4435-ad44-1c40c94e7a23)
3) wenn die Detail Ansicht geöffnet ist, wäre es sinnvoll/empfehlenswert die aktuellen Positionen in input Feldern angezeigt werden
3) wenn man die id verändert mit egal was, wird die Seite nicht aktualisiert